build/cross-image/Dockerfile

```diff
@@ -161,15 +161,24 @@ RUN GO_VERSION=1.7.3 && \
 ENV PATH /usr/local/go/bin:$PATH
 
 # precompile the go standard library for all supported platforms and configurations
+# there are three configurations that we compile for:
```

Comment:
does this change the typical dev flow invocation on our build?  we've generally been using `build/run make -j4 VERSION=dev CLIENT_SERVER_PLATFORMS=linux_amd64 release`, does that change at all?